Sprint-2/implement/lookup.js

```diff
@@ -1,5 +1,24 @@
-function createLookup() {
-  // implementation here
+
+function createLookup(countryCurrencyPairs) {
+  if (!Array.isArray(countryCurrencyPairs) || countryCurrencyPairs === null) {
```

Comment:
Is this check, `countryCurrencyPairs === null`, still necessary?

Reply:
!Array.isArray(countryCurrencyPairs)  check if the value countryCurrencyPairs is an array. i think we don't need this code countryCurrencyPairs === null